scripts/src/public/services/orders.ts

```diff
@@ -41,13 +42,14 @@ export interface Order {
 export interface MarketplaceOrder extends Order {
 	result?: AssetValue;
 	call_to_action?: string;
```

Comment:
1. put call_to_action on the top.
2. get rid of `MarketplaceOrder` and `ExternalOrder`
3. define result as `string | AssetValue`

Reply:
can you define a type JWT as `type JWT = string` and enjoy type safety for that?